freqtrade/optimize/backtesting.py

```diff
@@ -151,6 +151,7 @@ def cleanup():
         LoggingMixin.show_output = True
         PairLocks.use_db = True
         Trade.use_db = True
+        KeyValues.use_db = True  # ???
```

Comment:
What's the question here ?

Reply:
I don't understand the purpose of setting `use_db` to True in `backtesting:cleanup()`. Just didn't add up for me.

Reply:
These are set on the class - not on the instance.
If you don't reset it - half of (randomized) tests will fail - as it's set in the wrong state - and "non-backtesting" tests don't use the database.

Reply:
Oh I see now. Thanks for clarifying! Will cleanup this

Reply:
Addressed through https://github.com/freqtrade/freqtrade/pull/6908/commits/9fdb8b07accd04b4f629d075d52d6339e02ebaa2.